python/cli/prelude_cli/views/jobs.py

```diff
@@ -0,0 +1,28 @@
+import click
+
+from prelude_cli.views.shared import Spinner, pretty_print
+from prelude_sdk.controllers.jobs_controller import JobsController
+
+
+@click.group()
+@click.pass_context
+def jobs(ctx):
+    """ Jobs system commands """
+    ctx.obj = JobsController(account=ctx.obj)
+
+@jobs.command('statuses')
+@click.pass_obj
+@pretty_print
+def job_statuses(controller):
```

Comment:
there's also this one ha https://github.com/preludeorg/libraries/blob/master/python/cli/prelude_cli/views/detect.py#L315

maybe you can just scoop some the flags in that one and add them here?